style.css

```diff
@@ -33,5 +35,4 @@ button {
 	margin-left: 25px;
```

Comment:
Why do we need `margin-left` when we have a flex container with `justify-content: center;` ?